### ISSUE TITLE
Weblogo returning a html page

### ISSUE THREAD

**Issue by arupgsh** [biopython/biopython#4039]
With the current URL, the function is returning the HTML page itself(because of SSL redirect) and needs to be updated to "https://weblogo.threeplusone.com/create.cgi".

https://github.com/biopython/biopython/blob/2da0b26c294ad0fd63a45a3755e4a14042e2bc10/Bio/motifs/__init__.py#L506

**Comment by peterjc** [comment]
That looks like an easy fix - could you submit a pull request please?

**Comment by peterjc** [comment]
Thank you

**Comment by ivanerill** [comment]
This is still not working in the biopython 1.80 release, as far as I can see.

**Comment by peterjc** [comment]
The URL is HTTPS now. Can you save the HTML and look at it, perhaps there is an error message?

**Comment by ivanerill** [comment]
call: mymotif.weblogo("mymotif.png")
returns HTML file, no error message.
![mymotif](https://user-images.githubusercontent.com/1009065/213715277-91f69128-ee6b-467a-89eb-16a629c12065.png)

**Comment by peterjc** [comment]
When I rename that to something ending ``.html`` and view it, it looks like the weblogo form with an error message:

```html
<label class="err_msg">ERROR: Could not find Ghostscript on path. There should be either a gs executable or a gswin32c.exe on your system's path <br /><input style='float:right; font-size:small' type='submit' name='cmd_validate' value='Clear Error' /> </label>
```

I suspect right now the weblogo site itself is broken even if you ran a logo "by hand" via the website. Have you tried it?

**Comment by ivanerill** [comment]
right, sorry, did not see that the HTML was actually an error file.
https://weblogo.berkeley.edu/logo.cgi seems to be operating normally.

**Comment by peterjc** [comment]
Perhaps they've changed how the CGI works then, and the old Biopython code no longer works. Do you fancy investigating this?

**Comment by ivanerill** [comment]
but https://weblogo.threeplusone.com/create.cgi appears to be limited to EPS format currently

**Comment by ivanerill** [comment]
I will try to make direct calls to the API, see if I can get it working there and report back

**Comment by ivanerill** [comment]
```
from weblogo import *
input_file = open('mymotif.fas')
seqs = read_seq_data(input_file)
logodata = LogoData.from_seqs(seqs)
logooptions = LogoOptions()
logooptions.title = "A Logo Title"
logoformat = LogoFormat(logodata, logooptions)
eps = eps_formatter(logodata, logoformat)
png = logo_formatter.png_print_formatter(logodata, logoformat)
```

> OSError: Could not find Ghostscript on path. There should be either a gs executable or a gswin32c.exe on your system's path


reproduces the error, but only when invoking the `png `version. the first `eps` call runs and generates a valid logo, so I am assuming that this has to do with invoking ghostscript to generate the PNG from the EPS, and needs to fixed on their (https://weblogo.threeplusone.com/create.cgi) end? Or has the API behavior changed so that it now pushes the ghostscript conversion to the local user? On prior versions of the bio.motifs there was no need to have ghostcript locally installed to run weblogo.

**Comment by peterjc** [comment]
This does seem like a problem (or at least a change) on the website - have you tried PNG vs EPS there too?

**Comment by arupgsh** [comment]
In the current version of https://weblogo.threeplusone.com/create.cgi, all options except `EPS` and `Plain text` are disabled.

```
<select name="format">
<option disabled="disabled" value="png">PNG (low res.)</option>
<option disabled="disabled" value="png_print">PNG (high res.)</option>
<option disabled="disabled" value="jpeg">JPEG (low res.)</option>
<option selected="selected" value="eps">EPS (vector)</option>
<option disabled="disabled" value="pdf">PDF (vector)</option>
<option disabled="disabled" value="svg">SVG (vector)</option>
<option  value="logodata">Data (plain text)</option>
</select>
```

**Comment by cdanielmachado** [comment]
Exporting a logo in PNG format from the API returns what looks like a broken PNG file, but is actually an HTML file with an error message from the server that says "gs is not installed". I guess that server is not actively maintained or someone accidentally uninstalled ghostscript ? 

This server on the other hand works fine: https://weblogo.berkeley.edu/logo.cgi 

Would it be possible to simply replace one with the other?

**Comment by cdanielmachado** [comment]
Apologies, I just realized everything I said was already mentioned earlier in the thread... :)

**Comment by davidliuzw** [comment]
I'm currently using the latest release with the new URL. I'm afraid that this error still exists.

**Comment by peterjc** [comment]
@davidliuzw which file format are you asking for? Try EPS or plain text.

I guess #4049 isn't enough to close this. Perhaps the CGI URL needs to be an option, and we change the default as suggested by @ivanerill and @cdanielmachado to the Berkeley one?
